contribution/self-managed-k8s/crio/install-crio.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
```

Comment:
wondering if it is possible to use cri-o in self-managed-k8s-selinux?

Reply:
Haven't taken a look yet but will give it a try and update the script accordingly. Thanks!